src/falkordb-sentinel/sentinel-entrypoint.sh

```
@@ -1,74 +1,109 @@
-  TLS_CONNECTION_STRING="--tls --cacert $ROOT_CA_PATH"
-fi
+}
 
-SENTINEL_CONF_FILE=$DATA_DIR/sentinel.conf
-SENTINEL_LOG_FILE_PATH=$(if [[ $SAVE_LOGS_TO_FILE -eq 1 ]]; then echo $DATA_DIR/sentinel_$DATE_NOW.log; else echo ""; fi)
+initialize_runtime_paths() {
+  TLS_CONNECTION_STRING=$(if [[ $TLS == "true" ]]; then echo "--tls --cacert $ROOT_CA_PATH"; else echo ""; fi)
+  AUTH_CONNECTION_STRING="-a $ADMIN_PASSWORD --no-auth-warning"
+  DATE_NOW=$(date +"%Y%m%d%H%M%S")
+  SENTINEL_CONF_FILE=$DATA_DIR/sentinel.conf
+  SENTINEL_LOG_FILE_PATH=$(if [[ $SAVE_LOGS_TO_FILE -eq 1 ]]; then echo $DATA_DIR/sentinel_$DATE_NOW.log; else echo ""; fi)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)